### PR TITLE
Update: Add project Calico to the projects page

### PR DESCRIPTION
**Pull request by frozenprocess**
This PR adds the Project Calico link and logo
to the list of projects that are using eBPF!

Following the discussion in https://github.com/ebpf-io/ebpf.io-website/pull/166 , here is a rebased PR for Project Calico.

@borkmann  I didn't have the required permissions to rebase and force-push the old one.

### CODE REVIEW

File: src/data/major-projects.js
Comment (xmulligan) [inline src/data/major-projects.js:102]:
```suggestion
    title: "Networking and security for containers and Kubernetes",
```

File: src/data/major-projects.js
Comment (xmulligan) [inline src/data/major-projects.js:106]:
```suggestion
    description: `Calico Open Source is designed to simplify, scale, and secure 
    container and Kubernetes networks. Calico's eBPF dataplane utilizes the power 
    of eBPF to deliver networking, load-balancing, 
    and in-kernel security enforcement for your environment.`,
```

**Comment by xmulligan** [comment]
Thanks for the updated PR @frozenprocess. Just a few comments about wording to bring it in line with the rest of the page. First, everything on the page is open source so we don't need that in the description. Second, we prefer to have straight descriptions of the project.

My suggestions are just examples so feel free to edit if you prefer to describe it another way.

**Comment by frozenprocess** [comment]
@xmulligan Thanks for the review. I've changed the stuff that you pointed out let me know what you think.

TL/DR: Boy it is difficult to write these sort of stuff 😄

**Comment by xmulligan** [comment]
@dthaler Can you please review?

**Comment by frozenprocess** [comment]
Seems like website has gone through a lot of changes. I've updated my PR to get rid of conflicts.

@xmulligan  @dahavey Any updates on this one?

**Comment by frozenprocess** [comment]
@xmulligan, @borkmann similar to https://github.com/ebpf-io/ebpf.io-website/pull/166 this PR is headed for the same limbo again.

Since I've given a thorough explanation of [why/what/how](https://github.com/ebpf-io/ebpf.io-website/pull/166#issuecomment-1407609539) Calico eBPF should be included in the eBPF website would you be able to help and merge this request? or at least let me know what is the holdup and allow me to defend my claim?

**Comment by xmulligan** [comment]
@frozenprocess it is fine for me, I'm just wait for @dthaler to add his approval

**Comment by frozenprocess** [comment]
@xmulligan thanks for the reply. 
Please keep in mind that the original PR had [@dthaler's  approval](https://github.com/ebpf-io/ebpf.io-website/pull/166#pullrequestreview-883843728), and I only opened this one because I didn't have the required permission to rebase the [old one](https://github.com/ebpf-io/ebpf.io-website/pull/166#issuecomment-1410890847).

**Comment by xmulligan** [comment]
@frozenprocess Sorry for the delay on this. I just got confirmation from the BSC that they are happy to move forward with this PR. If you can update it, I'm happy to merge it

**Review by xmulligan — APPROVED** [review]

**Pull request merged**
